mkdocs.yml

```diff
@@ -0,0 +1,22 @@
+site_name: CZ Net Data Best Practices
+repo_name: cznethub/data-best-practices
+repo_url: https://github.com/cznethub/data-best-practices
+edit_uri: blob/develop/docs/
+
+theme: readthedocs
```

Comment:
The only thing I don't like about `readthedocs` theme is that I find the search very clunky

Try and compare with the material theme search:
https://cuahsi.github.io/mkdocs-example/sample_data/registering_samples/

It has search suggestions and highlighting
https://github.com/CUAHSI/mkdocs-example/blob/master/mkdocs.yml#L3-L7